hw3.py

```diff
@@ -0,0 +1,32 @@
+#3.
+tries = 1
+import random
+game = ["student", "homework", "vacation", "notebook", "pencil"]
+print("You have to guess the following word related to school.")
+a = random.choice(game)
+correct = a
+length = len(a)
+length = str(length)
+b = input("The word is " + length + " letters long. Guess a letter: ")
+for i in range(1, len(a)):
```

Comment:
what happens if the word has 5  or less characters? 
User should always has the right to enter 5 values, except is when word has less than 5 and user entered only correct values.